MANIFEST.in

```diff
@@ -0,0 +1,4 @@
+include pylintrc
+include LICENSE.txt
+include clean.sh
```

Comment:
The file `clean.sh` is not present, so no need to include it in the MANIFEST.in file. I did not check the other files.